plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -767,7 +767,7 @@ FROM
     rgwg.total_queued_request_count AS "Queued Request Count",
     rgwg.total_cpu_limit_violation_count AS "CPU Limit Violation Count",
     rgwg.total_cpu_usage_ms AS "CPU Usage (time)",
-    ' + CASE WHEN SERVERPROPERTY('ProductMajorVersion') > 10 THEN 'rgwg.total_cpu_usage_preemptive_ms AS "Premptive CPU Usage (time)",' ELSE '' END + '
+    ' + CASE WHEN SERVERPROPERTY('ProductMajorVersion') > 10 THEN 'rgwg.total_cpu_usage_preemptive_ms AS "Preemptive CPU Usage (time)",' ELSE '' END + '
```

Comment:
I will have a better look at it, but after a quick look, I've noticed that some of those corrections will actually change the existing measurements, by writing values to a different field key, or by changing a tag value. Of course, this could cause problems in reports or whatever is built on top of those data.

Reply:
I'd be happy to split off those (especially if you could identify them) into a distinct PR.

Reply:
If the field/tag key were spelled incorrectly, we aren't so strict that we would leave them even though strictly speaking it can cause issue with existing reports.  For example the `total_elasped_time_ms` -> `total_elapsed_time_ms` we ought to fix.

Could be good to identify the exact changes for the changelog or release notes though.  Let's split out the sqlserver.go file into a separate pr?